docs/docsite/rst/plugins/inventory.rst

```diff
@@ -31,7 +31,7 @@ If the plugin is in a collection and is not being picked up by the `auto` statem
    [inventory]
    enable_plugins = host_list, script, auto, yaml, ini, toml, namespace.collection_name.inventory_plugin_name
 
-Or, if it's a local plugin, perhaps stored in `~/.ansible/plugins/inventory/my_plugin.py`, you could reference it like this:
+Or, if it is a local plugin, perhaps stored in `~/.ansible/plugins/inventory/my_plugin.py`, you could reference it like this:
```

Comment:
i would not give this path but the 'setting' that cointains this and other paths  DEFAULT_INVENTORY_PLUGIN_PATH (link to it's doc page, which shows the defaults)